packages/wayland/compositor/sway/profile.d/04-sway.conf

```diff
@@ -0,0 +1,7 @@
+# SPDX-License-Identifier: GPL-2.0
+# Copyright (C) 2021-present Team LibreELEC (https://libreelec.tv)
```

Comment:
do we need a cpyr in a conf ?

Reply:
the Weston package et. al. used it so do I